src/realm.json

```diff
@@ -2250,13 +2251,41 @@
                 {
                     "authenticatorFlow": true,
                     "requirement": "REQUIRED",
-                    "priority": 31,
+                    "priority": 0,
                     "autheticatorFlow": true,
                     "flowAlias": "Authentication",
                     "userSetupAllowed": false
-                }
+                },
+                {
+                    "authenticatorFlow": true,
+                    "requirement": "REQUIRED",
+                    "priority": 1,
+                    "flowAlias": "Authorization",
+                    "userSetupAllowed": false,
+                    "autheticatorFlow": true
+                  }
+          
             ]
         },
+        {
+            "id": "7d53187d-13f4-4ec6-ade0-20b7d88d9aa4",
+            "alias": "Authorization",
```

Comment:
Not sure if it would belong in release here or in core (maybe both?) but would like to make sure we are tracking + documenting the manual upgrade addition of this flow to the realm (due to the current limitation around realm import and what pepr can manage).